src/main/java/ClientService.java

```diff
@@ -0,0 +1,66 @@
+import models.Client;
+import models.Phone;
+import java.util.Comparator;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.stream.Collectors;
+
+public class ClientService {
```

Comment:
It's not a service class, since it does not communicate with db and does not solve and bussiness tasks. It's better to find another name, but it's up to you